### ISSUE TITLE
Do not panic if one shot

### ISSUE THREAD

**Issue by Etsi0** [Globox1997/HerdsPanic#9]
### Minecraft Version

1.20.x

### Mod Loader Version

0.16.0

### Mod Version

1.1.0

### Describe the Issue

If you one shot a mob, the rest of the herd will not become panic

https://github.com/user-attachments/assets/52e8e441-b812-43bb-8564-53d295e8ee87

### latest.log

[latest.log](https://github.com/user-attachments/files/16644674/latest.log)

### Do you use any other mods except the required ones?

doesn't work even if herdpanic is the only mod installed

**Comment by Fyoncle** [comment]
Same here

**Comment by Fyoncle** [comment]
@Globox1997

**Comment by Globox1997** [comment]
This is how vanilla works :D
if you one shot a piglin, they won't get angry.

**Comment by Fyoncle** [comment]
> This is how vanilla works :D if you one shot a piglin, they won't get angry.

BRO :(

**Comment by Etsi0** [comment]
> This is how vanilla works :D if you one shot a piglin, they won't get angry.

That’s true, But it would be a pretty cool setting to add. It also makes more sense logically. if one of their friends gets taken out, the rest would start to panic and try to run away, even if it was a one-shot.

**Comment by Fyoncle** [comment]
> Author

Plus it is not how vanilla **works** but its more likely a bug so. We definitely want a toggle for it.
